openstates/tx/bills.py

```diff
@@ -259,30 +268,34 @@ def scrape_bill(self, session, history_url):
                     ctty = ctty.replace(r, "").strip()
                 extra['committees'] = ctty
 
-            bill.add_action(actor, action.findtext('description'),
-                            act_date, type=atype, **extra)
+            bill.add_action(
+                action.findtext('description'),
+                act_date,
+                chamber=actor,
+                classification=atype
+            )
```

Comment:
The three properties were `action_number`, `comment`, and `committee`. I couldn't find a likely home for `comment` in the [Bills or Action classes](https://github.com/opencivicdata/pupa/blob/master/pupa/scrape/bill.py), so I removed that code; we can restore it if there's a good spot to put it. `committee` can now be added as a related_entity to the action, so I did that. Since `action_number` was the only remaining item in the dictionary, I turned it into just a variable and kept its usage as-is. However, it did used to get passed along to the old call to `add_action`, so maybe the action has a place for it that I should be setting?